operator/pkg/clusterconfiguration/configuration_rpk.go

```diff
@@ -0,0 +1,63 @@
+package clusterconfiguration
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"gopkg.in/yaml.v3"
+)
+
+func NewRPKCfg(p *PodContext) *rpkCfg {
+	y := config.RpkYaml{
+		Version:    7,
+		Profiles:   []config.RpkProfile{config.DefaultRpkProfile()},
+		CloudAuths: []config.RpkCloudAuth{config.DefaultRpkCloudAuth()},
+	}
+	y.CurrentProfile = y.Profiles[0].Name
+	y.CurrentCloudAuthOrgID = y.CloudAuths[0].OrgID
+	y.CurrentCloudAuthKind = y.CloudAuths[0].Kind
+	return &rpkCfg{
+		PodContext: p,
+		RpkYaml:    &y,
+		fixups:     []Fixup{},
+	}
+}
+
+type rpkCfg struct {
+	*PodContext
+	*config.RpkYaml
+	fixups []Fixup
+}
+
+const (
```

Comment:
nit: consts should go at the top of the file.

Reply:
Ok, moved.